.github/header-checker-lint.yml

```diff
@@ -1,5 +1,8 @@
 allowedLicenses:
   - Apache-2.0
+allowedCopyrightHolders:
+  - Google
+  - OpenTelemetry Authors
```

Comment:
Are we in favor of adding this? I want the checker to pass so it remains a good signal, even though its not ideal to add it